src/id.md

```diff
@@ -1,8 +1,6 @@
 # Identity
 
-SORA allows you to store profile information in order to identify and discover people across the network. It is convenient for users to use the identity and registrar feature.
-
-A tutorial is provided below on how to set up an identity for yourself.
+SORA allows you to store profile information in order to identify and discover people across the network. It is convenient for users to use the identity and registrar feature. A tutorial is provided below on how to set up an identity for yourself.
```

Comment:
```suggestion
SORA allows you to store profile information in order to identify and discover people across the network. It's convenient for users to use the identity and registrar feature. A tutorial is provided below on how to set up an identity for yourself.
```

Reply:
@outoftardis this is a case where you would use it's :P

Reply:
I mean, those are not the same cases at all...